### ISSUE TITLE
ERROR: No such command 'checkpoint'. The command `checkpoint` does not exist.

### ISSUE THREAD

**Issue by nagocht** [fivetran/great_expectations#6072]
Hey,
I’m facing a weird error message in my GX installation.
```
Error: No such command 'checkpoint'.
The command `checkpoint` does not exist.
Please use one of: ['checkpoint', 'datasource', 'docs', 'init', 'project', 'store', 'suite']
The command `checkpoint` does not exist.
Please use one of: ['checkpoint', 'datasource', 'docs', 'init', 'project', 'store', 'suite']
Usage: great_expectations [OPTIONS] COMMAND [ARGS]...
Try 'great_expectations --help' for help.
```
I’m running GX in a docker container. The one deployed works just fine, if I execute GX locally it also works fine but running it in the local Docker container and deploying a new one results in this error message.
I’m using GX 0.14.0. I checked all the dependency differences between the working and not working versions but they seem to be the same. Using 0.14.2 doesn’t change the behaviour.

This is the Dockerfile I'm using:
```
FROM python:3.7

RUN apt clean
RUN apt-get update

RUN apt-get install make automake gcc g++

# Downloading gcloud package
RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz

# Installing the package
RUN mkdir -p /usr/local/gcloud \
    && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
    && /usr/local/gcloud/google-cloud-sdk/install.sh

# Adding the package path to local
ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin


WORKDIR /app

RUN pip install --upgrade pip
COPY requirements.txt /app/
RUN pip install -r requirements.txt 
COPY . /app

ENTRYPOINT [ "bash", "/app/entrypoint.sh" ] 
```

requirements.txt:
```
sqlalchemy==1.3.24
pybigquery==0.10.2
great_expectations==0.14.0
google-cloud-bigquery-storage==2.10.1
google-cloud-storage==1.44.0
pyarrow==6.0.1
jinja2==3.0.0 #using not the latest since it throughs an import error on contextfilter
```

Working Docker env:
```
HOSTNAME=<name>
PYTHON_VERSION=3.7.13
PWD=/app
PYTHON_SETUPTOOLS_VERSION=57.5.0
HOME=/root
LANG=C.UTF-8
GPG_KEY=<key>
TERM=xterm
SHLVL=1
PYTHON_PIP_VERSION=22.0.4
PYTHON_GET_PIP_SHA256=<>
PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/49ca29908cfd49683da12f2d5a4fa5689539f9d9/public/get-pip.py
PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/gcloud/google-cloud-sdk/bin
_=/usr/bin/env
```

Local Docker env:
```
Attaching to container
Activated service account credentials for: service account
SERVICE_JSON=<string>
HOSTNAME=<>
PYTHON_VERSION=3.7.14
SMTP_ADDRESS=smtp.sendgrid.net
SENDER_PASSWORD=<password>
RECEIVER_EMAILS=<email>
PWD=/app
GOOGLE_APPLICATION_CREDENTIALS=<json credential file>
PYTHON_SETUPTOOLS_VERSION=57.5.0
SENDER_ALIAS=<alias>
HOME=/root
LANG=C.UTF-8
GPG_KEY=<key>
SMTP_PORT=<port>
SHLVL=1
PYTHON_PIP_VERSION=22.0.4
SENDER_LOGIN=<login>
PYTHON_GET_PIP_SHA256=<>
PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/5eaac1050023df1f5c98b173b248c260023f2278/public/get-pip.py
PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/gcloud/google-cloud-sdk/bin
_=/usr/bin/env
```
The email configs and Google credentials are mapped in the docker compose file. Since I got the working docker config over ```docker run``` command , they're missing there.

I'm also attaching the pip lists for the working and not working docker env.
[local_docker.txt](https://github.com/great-expectations/great_expectations/files/9631531/local_docker.txt)
[docker_server.txt](https://github.com/great-expectations/great_expectations/files/9631532/docker_server.txt)

**Comment by austiezr** [comment]
Hey @nagocht ! Thanks for reaching out. We'll discuss internally and be in touch.

**Comment by magnitis** [comment]
Hi @nagocht , I have faced a similar issue with Great Expectations 0.14.6 on Ubuntu 20.04/Python 3.8.10. I was getting the same error with `checkpoint`, `datasource` and `suite` options (as far as I tested).

After debugging, I found the module `nbformat` was missing for some reason so the error was caused here: https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/render/renderer/datasource_new_notebook_renderer.py#L6.

`pip install nbformat` did the trick.

Not sure if that helps or is the best solution but I was able to continue working with no issues following that.
